openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -91,74 +91,80 @@ jQuery(function () {
             .then(module => module.initRealTimeValidation());
     }
     // conditionally load readmore button based on class in the page
     if (document.getElementsByClassName('read-more-button').length) {
-        import(/* webpackChunkName: "readmore" */ './readmore.js')
+        import (/* webpackChunkName: "readmore" */ './readmore.js')
             .then(module => module.initReadMoreButton());
     }
     // conditionally loads Goodreads import based on class in the page
     if (document.getElementsByClassName('import-table').length) {
-        import('./goodreads_import.js')
+        import ('./goodreads_import.js')
             .then(module => module.initGoodreadsImport());
     }
     // conditionally loads Related Carousels based on class in the page
     if (document.getElementsByClassName('RelatedWorksCarousel').length) {
-        import('./carousels_partials.js')
+        import ('./carousels_partials.js')
             .then(module => module.initCarouselsPartials());
     }
     // Enable any carousels in the page
     if ($carouselElements.length) {
-        import(/* webpackChunkName: "carousel" */ './carousel')
-            .then((module) => { module.init($carouselElements);
-                $('.slick-slide').each(function () {
+        import (/* webpackChunkName: "carousel" */ './carousel')
+            .then((module) => {
+                module.init($carouselElements);
+                $('.slick-slide').each(function() {
                     if ($(this).attr('aria-describedby') != undefined) {
-                        $(this).attr('id',$(this).attr('aria-describedby'));
+                        $(this).attr('id', $(this).attr('aria-describedby'));
                     }
                 });
             })
     }
     if ($('script[type="text/json+graph"]').length > 0) {
-        import(/* webpackChunkName: "graphs" */ './graphs')
+        import (/* webpackChunkName: "graphs" */ './graphs')
             .then((module) => module.init());
     }
 
     if (window.READINGLOG_STATS_CONFIG) {
-        import(/* webpackChunkName: "readinglog_stats" */ './readinglog_stats')
+        import (/* webpackChunkName: "readinglog_stats" */ './readinglog_stats')
             .then(module => module.init(window.READINGLOG_STATS_CONFIG));
     }
 
     const pageEl = $('#page-barcodescanner');
     if (pageEl.length) {
-        import(/* webpackChunkName: "page-barcodescanner" */ './page-barcodescanner')
+        import (/* webpackChunkName: "page-barcodescanner" */ './page-barcodescanner')
             .then((module) => module.init());
     }
 
     if (document.getElementById('modal-link')) {
-        import(/* webpackChunkName: "patron_metadata" */ './patron-metadata')
+        import (/* webpackChunkName: "patron_metadata" */ './patron-metadata')
             .then((module) => module.initPatronMetadata());
     }
 
+    if (document.getElementById('excerpts')) {
+        import ('./edit.js')
```

Comment:
You'll need to define a webchunk here `/* webpackChunkName: "patron_metadata" */`